.circleci/config.yml

```diff
@@ -35,6 +35,21 @@ workflows:
     jobs:
       - doc-gen-job
 
+  # Hourly build on weekdays
+  weekday:
+    triggers:
+      - schedule:
+          cron: "0 * * * 1-5"
```

Comment:
I'm fine with trying this out, but maybe hourly builds are a little too aggressive? Starting with some less frequent could also help us evaluate if this won't affect any resource usage limits on circleCI. Maybe every 4-6 hours? I don't have a strong opinion though.

Reply:
Sg, let me change it to every 4 hours then.